pkg/channel/README.md

```diff
@@ -0,0 +1,26 @@
+# Eventing-Kafka Channel
+
+The "Channel" implementation is a Kafka Producer which is responsible for
+receiving CloudEvents, converting them to Kafka Messages, and writing them
+to the appropriate Kafka Topic.
+
+A unique Channel Deployment / Service is created for every Kafka Secret (in
+the knative-eventing namespace and labelled with
+`knativekafka.kyma-project.io/kafka-secret: "true"`).  The single Deployment
```

Comment:
It would be nice to have all those references updated before merging (there are too many for me to comment on), since the project is now part of Knative.

Reply:
Oops, missed those! I'll fix them